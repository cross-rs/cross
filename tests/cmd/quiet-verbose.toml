bin.name = "cross"
args = ["build", "--quiet", "--verbose"]
status = "failed"
