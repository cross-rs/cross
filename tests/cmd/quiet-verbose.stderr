[..][cross] error[..]:[..] cannot set both --verbose and --quiet
